use-cases/tls.md

```diff
@@ -11,7 +11,7 @@ You can store TLS-related metadata under `tls.`, when appropriate.
 | [source.ip](../README.md#source.ip)  | IP address of the source.<br/>Can be one or multiple IPv4 or IPv6 addresses. | core | ip | `10.1.1.10` |
 | [destination.ip](../README.md#destination.ip)  | IP address of the destination.<br/>Can be one or multiple IPv4 or IPv6 addresses. | core | ip | `5.5.5.5` |
 | [destination.port](../README.md#destination.port)  | Port of the destination. | core | long | `443` |
-| <a name="tls.version"></a>*tls.version* | *TLS version.* | (use case) | keyword | `TLSv1.2` |
+| [tls.version](../README.md#tls.version)  | TLS version. | extended | keyword | `TLSv1.2` |
```

Comment:
I don't know what this means, but I'll ignore it as directed ;-)

Reply:
Yes, nobody knows what the "use cases" are in the repo, anymore.

The folks that find them get confused by them, because we haven't been able to maintain them for a long time.